css/tutorial.styl

```diff
@@ -25,10 +25,14 @@
     > .media-card-content
       @extends .content-container
 
+    img
+      max-width: 100%
```

Comment:
I noticed that `media-card-media` already had that attribute, but I was noticing the error on some tutorials, particularly the one for Amazon Aerobotany. This CSS addition appeared to fix the issue.